### ISSUE TITLE
Investigate OpenSubtitles Addon Issues

### ISSUE THREAD

**Issue by jaruba** [Stremio/stremio-opensubtitles#19]
I've seen this happen quite a bit lately, and other users have seen similar strange behaviour.

The issue:
- the addon works in the Desktop app, but not the Android app (gives the "Error Fetching Subtitles" error)
- the other case has also been reported: working on Android, but not on Windows

The issues usually resolve themselves after some time.

When this was reproduced once (and it worked on Desktop but not Android) the OpenSubtitles link for that stream loaded fine in Chrome, even with a cache break.

Cloudflare should be checked, the OpenSubtitles addon server should be checked for logs and we should make sure that this is not some load issue.

**Comment by Amzd** [comment]
I am seeing different results in web app as on local windows app, running on the same machine.

Web result with only one language available:
![image](https://user-images.githubusercontent.com/17001151/209661874-e5d12060-a249-4fce-b284-5b6596755610.png)

App:
![image](https://user-images.githubusercontent.com/17001151/209662074-9045ef5d-492e-4c1f-937e-ec2abd89fd48.png)

**Comment by jaruba** [comment]
@Amzd I don't believe that this is caused by the same issue that I created, and your comment does not include enough details to pinpoint the issue you are facing...

If you could give more details for the purpose of reproducing it, we can attempt to fix it.

**Comment by Amzd** [comment]
Should I create a new issue? How can I give you more details? I have the latest stremio with this add-on and a torrent add-on. When I stream through the app I get many subtitle options but when streaming from http://127.0.0.1:11470 I get none or only one. I suspect it is not querying for the same stream? But I don't know how to inspect the queries in the app.

**Comment by jaruba** [comment]
@Amzd So you are using Stremio with [https://app.strem.io/shell-v4.4/](https://app.strem.io/shell-v4.4/) and the Stremio app is still running while using it (because it needs the streaming server to play torrents) and you are getting fewer subtitle options from the opensubtitles addon? I am unable to reproduce this with the following:
<img width="121" alt="Screenshot 2022-12-27 at 16 37 30" src="https://user-images.githubusercontent.com/1777923/209681744-bcfecca1-c5e0-43cd-a5bf-d5c60a58b237.png">

**Comment by Amzd** [comment]
That is correct. With the exact same version. 
![image](https://user-images.githubusercontent.com/17001151/209684836-1b400abd-35a6-4eef-b54b-ac1276fd270f.png)

**Comment by jaruba** [comment]
@Amzd Ah, that is expected, that Stremio endpoint does not support embedded subtitles, they are lost during the transcoding process, it will work when we officially release the Stremio Web project.

**Comment by Amzd** [comment]
Ah that is too bad. Where can I help make this a reality?
